### DISCUSSION TITLE
Input type="date" includes a date picker. Any way to style this?

### DISCUSSION THREAD

**Discussion by LoneE-PP** [heroui-inc/heroui#1698] [Q&A]
Hope someone can help in styling at least some colors of the datepicker? 
![image](https://github.com/nextui-org/nextui/assets/74532864/949b138c-03c4-4b13-a700-a7fbc52ab9c0)

**Comment by caudurodev** [comment]
What you are seeing is the [browser html date picker.](https://dev.to/codeclown/styling-a-native-date-input-into-a-custom-no-library-datepicker-2in)

There are some options for styling this but it's not really well supported. There is a feature request for NextUI to add a datepicker [here](https://github.com/nextui-org/nextui/discussions/1537)
